### PR TITLE
Tabs -> spaces (part of #166)

### PR DESCRIPTION
**Pull request by beatgammit**
I used Dart Editor when prepping #165, and that did automatic tabs -> spaces conversion (which I manually fixed before submitting). I think fixing this will make diffs merged before `dartfmt` is run cleaner.

In case it matters, to create this diff, I ran (on Linux):

    find -name '*.dart' -exec sed -i 's:	:  :g' {} \;

There may be some oddness if any of the code was edited assuming a tab width != 4, but that should be fixed by `dartfmt`.

### CODE REVIEW

**Comment by beatgammit** [comment]
As far as I can tell, this doesn't affect any current pull requests (thankfully).

**Comment by robsilv** [comment]
Thanks @beatgammit. I'm a bit out of the loop on Dart formatting guidelines - are spaces recommended over tabs? Are you saying this is the default behaviour of the editor, so most likely everyone's code will be spaces?

**Comment by beatgammit** [comment]
@robsilv Yes, here's the [style guide](https://www.dartlang.org/articles/style-guide/).

TL;DR - 2-space indenting, no tabs

**Comment by robsilv** [comment]
Ok, done.
